### ISSUE TITLE
Does not support wcag21aa

### ISSUE THREAD

**Issue by zenntenn** [TroyWalshProf/SeleniumAxeDotnet#240]
**Describe the bug**
There may be issues with other tags as well, but when I use the tags wcag21aa, wcag21a, or wcag2aa I'm not getting violations, on pages that fail with plenty of violations in wcag2a.

**To Reproduce**
Run aXe core tests against https://cenw-wscoe.github.io/practice-site/banana/
Use the tag wcag21aa. 
The AxeResults for .Passes, .Violations, .Incomplete, and .Error are all empty.

**Expected behavior**
AxeResults .Violations should contain many violations.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: happens with Edge, Chrome, and Firefox
 - Version [e.g. 22]: 3.1.3

**Comment by TroyWalshProf** [comment]
What does your AxeBuilder code look like?
Note* As far as I can tell, the practice site has wcag21a failure but no wcag21aa issue.

**Comment by TroyWalshProf** [comment]
@zenntenn - Could you provide some sample code for reproducing this?

**Comment by zenntenn** [comment]
@TroyWalshProf 

```
var options = new ChromeOptions();
options.AddArgument("--headless");
options.AcceptInsecureCertificates = true;
options.AddArguments("--disable-gpu");
options.AddArguments("--no-sandbox");
options.AddArguments("--disable-extensions");
var driver = new ChromeDriver(ChromeDriverService.CreateDefaultService(), options, TimeSpan.FromMinutes(3));
driver.Manage().Window.Maximize();
driver.Navigate().GoToUrl("https://cenw-wscoe.github.io/practice-site/banana");
_ = Wait.Until(wd => ((IJavaScriptExecutor)driver).ExecuteScript("return document.readyState").ToString() == "complete");
var builder = new AxeBuilder(driver);
var results = builder.WithTags(AXeTag).Analyze();
using var scope = new AssertionScope();
scope.FormattingOptions.MaxDepth = 100;
scope.FormattingOptions.MaxLines = 10000;

results.Violations.Should().BeEmpty();
results.Incomplete.Should().BeEmpty();
results.Error.Should().BeNullOrEmpty();
results.Passes.Should().NotBeEmpty();
```

**Comment by TroyWalshProf** [comment]
Are you 100% sure that page has wcag2aa violations?
Tested the site with the Chrome Axe extension and the closest I found were wcag2a violations.

**Comment by zenntenn** [comment]
For violations a is always a subset of aa violations (and aa is a subset of
aaa violations).

**Comment by TroyWalshProf** [comment]
I am guessing you will need to bring these issues up in: https://github.com/dequelabs/axe-core 
*This project uses Axe Core under the hood.

As far as the wcag21a thing goes, the closest I see is wcag2a.

**Comment by zenntenn** [comment]
It looks like axe core added at least some support for wcag2.1a back in
2018, or at the very least returns the same results as wcag2a
https://www.deque.com/blog/support-for-wcag-2-1-in-axe-core/

Going through the axe core GitHub seems to support that it has wcag 2.1

**Comment by TroyWalshProf** [comment]
Using Axe core directly I was unable to find and wcag 2.1 violations on the test page. 
If it helps here is Deque's (the folks that build and maintain Axe Core) chrome extension:
https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US

Notes* 
The C# and Chrome extension should produce the same result. 
The more you share the more it sounds like this issue should be raise in the Axe Core project.

**Comment by zenntenn** [comment]
Ok, thanks, I'll go down that route then.

**Comment by TroyWalshProf** [comment]
@zenntenn - Unless there is a strong objection, I will closing this bug.

**Comment by zenntenn** [comment]
Go ahead and close
